### ISSUE TITLE
tabel date range

### ISSUE THREAD

**Issue by Wappie2000** [SteefP/myHomeBank#10]
Ik zou als je output pakt altijd de datum range pakken van de gebruikte set.

**Comment by SteefP** [comment]
yes ik weet het. Probleem is een beetje dat ik nu om functionaliteit toe te voegen eerst mijn oude code moet  her indelen. "de plant moet gesnoeid worden om te groeien:)" maar o  iddd handig zijn aals er gerefactort gaaat worden  komt dit ook meteen aan bod

**Comment by Wappie2000** [comment]
Lijkt me geen haast te hebben, maar dan zou ik de date range leeg maken om te beginnen. De waardes nu kunnen voor verwarring zorgen.
